examples/introExample/build.gradle

```diff
@@ -1,13 +1,17 @@
+apply plugin: 'android-sdk-manager'
 apply plugin: 'com.android.application'
 apply plugin: 'android-command'
+apply plugin: 'com.neenbedankt.android-apt'
```

Comment:
Just testing that it still works when the apt plugin is also installed manually by users.